pkg/serviceaccount/claims.go

```diff
@@ -181,35 +183,37 @@ func (v *validator) Validate(ctx context.Context, _ string, public *jwt.Claims,
 	noderef := private.Kubernetes.Node
 	secref := private.Kubernetes.Secret
 	// Make sure service account still exists (name and UID)
-	serviceAccount, err := v.getter.Cluster(clusterName).GetServiceAccount(namespace, saref.Name)
```

Comment:
Diff here is amobinantion. It basically adds `if v.lookup` to two lookups. It shows somehow strange...